mobile_config/src/sub_dao_epoch_reward_info.rs

```diff
@@ -0,0 +1,113 @@
+use crate::EpochPeriod;
+use chrono::{DateTime, Utc};
+use file_store::traits::{TimestampDecode, TimestampEncode};
+use helium_proto::services::sub_dao::SubDaoEpochRewardInfo as SubDaoEpochRewardInfoProto;
+use rust_decimal::prelude::*;
+use std::ops::Range;
+
+#[derive(Clone, Debug)]
+pub struct ResolvedSubDaoEpochRewardInfo {
+    pub epoch: u64,
```

Comment:
Would it be beneficial to name this something like `epoch_day`? Almost everywhere else we use `epoch` it's in reference to a timestamp or Range. 

Or would it make sense to have `EpochPeiod` at this level?

Reply:
addressed here: e8643f61bafa27c8c1a9680da8f625f068f01393